### PR TITLE
Marsh add effects t3

### PR DESCRIPTION
**Pull request by braydenmarsh**
## Task 3 - Add Effects to Vocals in Logic Pro

**Completed Work**

- Created new procedure: `marshb/marsh-logic-add-effects-T3.md`
- Procedure's Goal: Instruct experienced user on how to apply vocal affects in a logical way.

## User Scenario

John has some experience with vocal production but he wants to be more creative and unique with his voice. He has been using vocal presets but wants more control over his vocal chain. He needs instructions on where to add effects so that he can experiment with different production styles.

## Notable Review Requests

I am going to add graphics for the first section of this document. At the present moment I am unable to do so because of technology problems. 

For my second section, do you believe that my paragraphs are too wordy? Is the block of text an unefficient way of teaching the user about vocal chains?

Is the organization of my document done well? Where do you believe I could improve?

### CODE REVIEW

**Comment by lhmarlie** [comment]
hey Brayden, I commented on your other two branches but missed this one - sorry!

-  the graphics added in the first section are helpful but you might want to explain the difference between the red and green boxes or the chronology of them (which box should the user concern themselves with first)
- for the second section, I like the breakdown of term and then definition. I think aesthetically it looks the best and breaks up the knowledge into small, consumable bits
- for organization, overall I think you are good. One recommendation might be to have a link at the beginning of the procedure to the definitions list (I'm not sure how to do this but it could be useful for a user to see right at the beginning that you have included a list instead of only scrolling to find it)

**Comment by Eden-Miller** [comment]
Hey Brayden!

I know this is too late to help you with your initial submission, but maybe it can help you during the resubmission process (if you decide to do so) or just for future reference. I'd like to thank you again for being so flexible and understanding of my circumstances. 

- I like your hyperlinked options to go to additional documentation if needed. Some people the goal is to keep the reader on the page as long as possible, but in this case I think the hyperlinks do a good job at showing the information ecosystem. 
- You staging steps are good. You initial tagline gives a good example of what vocal effects **Logic Pro** can do. It does leave me confused as to what **Logic Pro** is. Are we assuming your audience is already somewhat familiar with the software? Conditions work as a good prerequisite. I would consider rewording "own a..." maybe "have access to"?
- Your coaching moves are good! I would consider adding arrows or numbering the steps someone should take in your screenshots. 
- Alerting moves are clearly marked. 
- I think your last section is interesting. It functions as a reference topic. I'm unsure if I would include that in the initial procedure, but the information is a good addition. That's more of a Google or Dr. Lindgren question about procedure genre conventions. 

I hope this can help you somewhat! Please feel free to reach out if you have any questions or want additional feedback!
